elki/src/main/java/de/lmu/ifi/dbs/elki/algorithm/itemsetmining/DenseItemset.java

```
@@ -68,6 +74,10 @@ public boolean containedIn(BitVector bv) {
   public long[] getItems() {
     return items;
   }
+  
+  public int[] toSparseRep() {
+    return new int[0];
```

Rather throw an exception than returning an incorrect value!